client/wfnews-war/src/main/angular/src/app/services/report-of-fire-service.ts

```diff
@@ -93,6 +93,18 @@ formData.append('image3', await this.convertToBase64(image3));
 return;
 }
 
+      const storedOfflineReportData = await this.storage.get('offlineReportData');
```

Comment:
awaits from promises should generally have a catch, as right now any exception occurring in the promise may not be handled or caught properly